src/lib.rs

```diff
@@ -515,15 +515,30 @@ impl Debug for Ext4 {
     }
 }
 
+#[cfg(feature = "std")]
+#[cfg(test)]
+fn load_test_disk() -> Ext4 {
+    let output = std::process::Command::new("zstd")
+        .args([
+            "--decompress",
+            // Write to stdout and don't delete the input file.
+            "--stdout",
+            "test_data/test_disk1.bin.zst",
+        ])
+        .output()
+        .unwrap();
+    assert!(output.status.success());
+    Ext4::load(Box::new(output.stdout)).unwrap()
+}
```

Comment:
This is so fast we don't want to do any caching or anything?

Reply:
Right, the tests complete in two seconds or so, so doesn't seem worth it. Added a comment.